src/test/java/com/microsoft/sqlserver/jdbc/databasemetadata/DatabaseMetadataGetIndexInfoTest.java

```diff
@@ -0,0 +1,174 @@
+package com.microsoft.sqlserver.jdbc.databasemetadata;
+
+import static org.junit.jupiter.api.Assertions.*;

```

Comment:
Use specific imports.

```suggestion
import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertNull;
import static org.junit.jupiter.api.Assertions.assertTrue;
import static org.junit.jupiter.api.Assertions.fail;
```